fluentui_drawer/src/main/java/com/microsoft/fluentui/tokenized/drawer/Drawer.kt

```diff
@@ -202,11 +264,18 @@ class DrawerState(
  */
 @Composable
 fun rememberDrawerState(confirmStateChange: (DrawerValue) -> Boolean = { true }): DrawerState {
-    return rememberSaveable(saver = DrawerState.Saver(confirmStateChange)) {
-        DrawerState(DrawerValue.Closed, confirmStateChange)
+    return rememberSaveable(saver = DrawerState.Saver(expandable = true, skipOpenState = false, confirmStateChange = confirmStateChange)) {
+        DrawerState(initialValue = DrawerValue.Closed, expandable = true, isSkipOpenState = false, confirmStateChange)
+    }
+}
+@Composable
+fun rememberBottomDrawerState(initialValue: DrawerValue = DrawerValue.Closed, confirmStateChange: (DrawerValue) -> Boolean = { true }, expandable: Boolean = true, skipOpenState: Boolean = false ): DrawerState {
```

Comment:
put confirmStateChange param at the end

Reply:
don't expose "initialValue". It would throw error if user set it to expanded state but expanded state is not available for content size having less than half of screen.

Reply:
I have unexposed initialValue, and put confirmStateChange param at the end.